lib/css/modules/_showImages.scss

```diff
@@ -10,16 +10,25 @@ img {
 	image-orientation: from-image;
 }
 
+%expando-button-overlay {
+	display: inline-block;
+	content: ' ';
+	width: 100%;
+	height: 100%;
+	pointer-events: none;
+}
+
 .expando-button {
 	cursor: pointer;
 	user-select: none;
 
+	.res-showImages-highlightSpoilerButton .thing.spoiler &::before {
+		@extend %expando-button-overlay;
+		background-image: linear-gradient(45deg, rgba(255, 130, 0, .30) 25%, transparent 25%, transparent 50%, rgba(255, 130, 0, .30) 50%, rgba(255, 130, 0, .30) 75%, transparent 75%, transparent);
```

Comment:
NSFW styling should override spoiler IMO. Or should we add a combo rule to show both stripes at once?